kafka-streams-serdes/src/main/java/org/hypertrace/core/kafkastreams/framework/serdes/proto/ProtoSerializer.java

```diff
@@ -0,0 +1,11 @@
+package org.hypertrace.core.kafkastreams.framework.serdes.proto;
+
+import com.google.protobuf.Message;
+import org.apache.kafka.common.serialization.Serializer;
+
+public class ProtoSerializer<T extends Message> implements Serializer<T> {
```

Comment:
Can you please add a documentation comment explaining why this is required and how is it different from the built-in one?